can't build vw cluster

Hi,
It seams that now I can't build current master branch from scratch with just make (without calling ./autogen.sh and libtool deps) due to problems with cluster step.  
The log is here http://pastebin.com/0YGe3GfF  
Some bits of it:
```
...
/usr/bin/g++ -std=c++0x   -D__extern_always_inline=inline -Wall -pedantic -O3 -fomit-frame-pointer -fno-strict-aliasing  -D_FILE_OFFSET_BITS=64 -DNDEBUG -I /usr/include  -fPIC  -o vw main.o -L. -l vw -l allreduce -L /usr/lib -L /usr/lib/x86_64-linux-gnu -l boost_program_options -l pthread -l z
make[1]: Leaving directory '/home/truf/ml/vowpal_wabbit_current/vowpalwabbit'
cd cluster; make
make[1]: Entering directory '/home/truf/ml/vowpal_wabbit_current/cluster'
/usr/bin/g++ -std=c++0x   -D__extern_always_inline=inline -Wall -pedantic -O3 -fomit-frame-pointer -fno-strict-aliasing  -D_FILE_OFFSET_BITS=64 -DNDEBUG -I /usr/include  -fPIC  -I ../vowpalwabbit -std=c++0x  -c spanning_tree_main.cc -o spanning_tree_main.o
/usr/bin/g++ -std=c++0x   -D__extern_always_inline=inline -Wall -pedantic -O3 -fomit-frame-pointer -fno-strict-aliasing  -D_FILE_OFFSET_BITS=64 -DNDEBUG -I /usr/include  -fPIC  -I ../vowpalwabbit -std=c++0x  -c ../vowpalwabbit/spanning_tree.cc -o ../vowpalwabbit/spanning_tree.o
/usr/bin/g++ -std=c++0x   -D__extern_always_inline=inline -Wall -pedantic -O3 -fomit-frame-pointer -fno-strict-aliasing  -D_FILE_OFFSET_BITS=64 -DNDEBUG -I /usr/include  -fPIC  -I ../vowpalwabbit -std=c++0x  -o spanning_tree spanning_tree_main.o ../vowpalwabbit/spanning_tree.o ../vowpalwabbit/vw_exception.o 
../vowpalwabbit/spanning_tree.o: In function `VW::SpanningTree::Start()':
spanning_tree.cc:(.text+0x1af1): undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
Makefile:26: recipe for target 'spanning_tree' failed
make[1]: *** [spanning_tree] Error 1
make[1]: Leaving directory '/home/truf/ml/vowpal_wabbit_current/cluster'
Makefile:90: recipe for target 'spanning_tree' failed
make: *** [spanning_tree] Error 2

```
Moreover running ./autogen.sh enforces clang usage as was described in this problem: https://github.com/JohnLangford/vowpal_wabbit/issues/678
Build log after autogen usage: http://pastebin.com/SbZnGNu8
I would stick with gcc by default and no libtool. It was so convenient to use just 'make/make test' before. Could this be adjusted?

--- a/cluster/Makefile
+++ b/cluster/Makefile
@@ -23,7 +23,7 @@ all:	spanning_tree
 	$(CXX) $(FLAGS) -c $< -o $@
 
 spanning_tree: spanning_tree_main.o ../vowpalwabbit/spanning_tree.o ../vowpalwabbit/vw_exception.o
-	$(CXX) $(FLAGS) -o $@ $+ 
+	$(CXX) $(FLAGS) -o $@ $+ $(LIBS) 
 
 install: spanning_tree
 	cp spanning_tree /usr/local/bin

